ui/imports/shared/popups/SettingsDirtyToastMessage.qml

```diff
@@ -148,13 +148,12 @@ Rectangle {
             onClicked: root.saveForLaterClicked()
         }
 
-        DisabledTooltipButton {
+        StatusButton {
             id: saveChangesButton
             objectName: "settingsDirtyToastMessageSaveButton"
-            buttonType: DisabledTooltipButton.Normal
             text: root.defaultSaveChangesText
-            enabled: root.active && root.saveChangesButtonEnabled
             interactive: root.active && root.saveChangesButtonEnabled
+            // TODO tooltip when !interactive
```

Comment:
I think we wanted to have some kind of tooltip here when the save is not possible? @noeliaSD @alexjba

Reply:
Ah, I get it now... will fix it :)